excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/SpreadsheetParser.java

```diff
@@ -22,11 +22,12 @@
     private static final Logger logger = LoggerFactory.getLogger(SpreadsheetParser.class);
```

Comment:
I wanted to add a test for this, but this class at the moment hasn't got any and it's quite untestable as it is. I would need to refactor it to allow testability. If you think we need to do this I'm going to do that.